documentation/installation/kubernetes.md

```
@@ -212,6 +212,33 @@ redis-dev-backend-redis-68c9f66786-4dgxj   1/1     Running   0          4m21s
 
 ```
 
+### Find ICAP server connection values
+
+Docs can scan every uploaded file with an antivirus engine exposed through the
+Internet Content Adaptation Protocol (ICAP), see
+[malware detection](../malware_detection.md). For development, an ICAP server
+(c-icap with ClamAV) is provided by the dev dependencies chart, so you can
+install a local testing environment as follow:
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)